docker-compose.platform.yml

```diff
@@ -45,7 +45,7 @@ services:
       - 26656:26656 # P2P
     volumes:
       - ./configs/${NETWORK:?err}/drive/tendermint:/tendermint/config
```

Comment:
You can't mount file into already mounted dir (`drive_tendermint:/tendermint`) so I would bring back the previous separation of data and config dirs. To do so you need to revert tendermint config and mount data dir to named volume (`drive_tendermint_data:/data`)

Reply:
Actually this works fine, in the same way as it now works for Dash Core if using the correctly prepared image. All you need to do is make sure the dirs exist in the image with the correct owner before mounting anything. It looks like this inside the running container:
```
bash-4.4$ ls -lha
total 32
drwxr-sr-x    5 tmuser   tmuser      4.0K Aug 25 07:49 .
drwxr-xr-x    1 root     root        4.0K Aug 25 07:49 ..
drwxrwxr-x    2 1000     tmuser      4.0K Aug 25 07:14 config
drwx--S---    2 tmuser   tmuser      4.0K Aug 25 07:49 data
drwxr-sr-x    4 tmuser   tmuser      4.0K Aug 25 07:49 db
-rw-------    1 tmuser   tmuser       148 Aug 25 07:49 node_key.json
-rw-------    1 tmuser   tmuser       345 Aug 25 07:49 priv_validator_key.json
-rw-------    1 tmuser   tmuser        48 Aug 25 07:49 priv_validator_state.json
bash-4.4$ ls config -lha
total 40
drwxrwxr-x    2 1000     tmuser      4.0K Aug 25 07:14 .
drwxr-sr-x    5 tmuser   tmuser      4.0K Aug 25 07:49 ..
-rw-rw-r--    1 1000     tmuser     13.5K Aug 25 07:14 config.toml
-rw-rw-r--    1 1000     tmuser     13.5K Aug 21 00:01 genesis.json
bash-4.4$ 
```

Reply:
Excellent! 👍

Reply:
Are you sure those configs are the same? :))